app/assets/stylesheets/components/_verify.scss

```diff
@@ -0,0 +1,6 @@
+@use '../variables/app' as *;
+
+.styled-as-h4 {
+  font-size: $h4;
```

Comment:
It looks like there's already an `.h4` class defined as such:
```
h4,
.h4 {
  @extend %h4;
}
```

Might be easier to apply that than creating a new component here?

Reply:
Thanks for pointing that out. I hadn't realized there was a class for that.

Unfortunately I run into the same spacing issue that I added the `inline` piece for.

Here's using the `h4` class:

<img width="468" alt="h4-class" src="https://github.com/18F/identity-idp/assets/35475380/d35dc973-2138-432e-bdbd-b29e81aa8d14">

<hr>

Here's using the `styled-as-h4` class:

<img width="483" alt="styled-as-h4-class" src="https://github.com/18F/identity-idp/assets/35475380/0593220e-62f0-4d63-b5e1-b9effbcb870e">

<hr>

It's maybe most noticeable looking at the gap between `Current residential address` and the line above it.

Reply:
Gotcha. Without looking more at the HTML/CSS I'm not sure what's going on. But there is an existing `display` utility class in [USWDS](https://designsystem.digital.gov/utilities/display/) that you can just add to the markup that should help. I think the class is just `.display-inline`